docs/additional-functionality/advanced_configs.md

```diff
@@ -386,6 +386,8 @@ Name | SQL Function(s) | Description | Default Value | Notes
 <a name="sql.expression.ToUnixTimestamp"></a>spark.rapids.sql.expression.ToUnixTimestamp|`to_unix_timestamp`|Returns the UNIX timestamp of the given time|true|None|
 <a name="sql.expression.TransformKeys"></a>spark.rapids.sql.expression.TransformKeys|`transform_keys`|Transform keys in a map using a transform function|true|None|
 <a name="sql.expression.TransformValues"></a>spark.rapids.sql.expression.TransformValues|`transform_values`|Transform values in a map using a transform function|true|None|
+<a name="sql.expression.TruncDate"></a>spark.rapids.sql.expression.TruncDate|`trunc`|Truncate the date to the unit specified by the given string format|true|None|
```

Comment:
> Returns timestamp `ts` truncated to the unit specified by the format model `fmt`.

I suppose that we support the same string format as spark? We should call it out what string format we support, like in [the spark doc](https://spark.apache.org/docs/3.5.3/sql-ref-functions-builtin.html#date-and-timestamp-functions). The above is a snippet from their doc.

Reply:
Is this a feature flag? That means, `date_trunc` will run on the GPU if this config is true, but not otherwise? This doc seems quite confusing to me, especially the `Default Value` column. Should it be rather called `Supported by default`?

Reply:
These are automatically generated by `mvn verify`, so reading it https://nvidia.github.io/spark-rapids/docs/additional-functionality/advanced_configs.html would be `true|None => Default Value | Note`.

Reply:
> We should call it out what string format we support, like in [the spark doc](https://spark.apache.org/docs/3.5.3/sql-ref-functions-builtin.html#date-and-timestamp-functions). The above is a snippet from their doc.

That is just a short description of the operator like in https://nvidia.github.io/spark-rapids/docs/additional-functionality/advanced_configs.html#expressions. I don't think it is where we put the full description.

Reply:
Hmm OK. Where is the full description then? Do we just suggest to look at the spark docs for the full description?

Reply:
I think so, as here we only overload Spark operators instead of implementing new ones. We typically give more descriptions if our overloaded versions have something different than those of Spark.

Reply:
OK, agree that the full description is not necessary here. Though some links to the full description would help the readers to find the full description easily. It should be done in a separate PR though if we ever add them.